pkg/security/serializers/serializers_linux.go

```diff
@@ -1070,15 +1069,14 @@ func newDDContextSerializer(e *model.Event) *DDContextSerializer {
 	for ptr != nil {
 		pce := (*model.ProcessCacheEntry)(ptr)
 
-		if pce.SpanID != 0 || pce.TraceID != 0 {
-			s.SpanID = pce.SpanID
-			s.TraceID = pce.TraceID
+		if pce.SpanID != 0 && pce.TraceID.Hi != 0 && pce.TraceID.Lo != 0 {
```

Comment:
Shouldn't the condition be:
```
pce.SpanID != 0 && (pce.TraceID.Hi != 0 || pce.TraceID.Lo != 0)
```
?

Reply:
The specification defines: `TraceId A valid trace identifier is a 16-byte array with at least one non-zero byte.`
So I guess a TraceID having its firsts 8 bytes set to 0 is still a valid one, that's why I check that all 16 bytes are nil.

Reply:
after discussion, you're right I'll make the change, thank you!